bzt/modules/console.py

```diff
@@ -128,7 +129,10 @@ def prepare(self):
             modules += self.engine.provisioning.executors
         for module in modules:
             if isinstance(module, WidgetProvider):
-                widgets.append(module.get_widget())
+                widget = module.get_widget()
+                widgets.append(widget)
+                if isinstance(widget, SidebarWidget):
```

Comment:
Because we have Monitoring() with MonitoringWidget among WidgetProvider's, and MonitoringWidget doesn't have update() method